services/visual_recognition/v3.js

```diff
@@ -265,7 +224,19 @@ module.exports = function (RED) {
             params['owners']=msg.params['owners'];
           if ( msg.params != null && msg.params.threshold != null)
             params['threshold']=msg.params['threshold'];
-          performAction(params, feature, actionComplete);
+          
+          switch(feature)
+          {
+            case 'classifyImage' : 
+              visualRecognition.classify(params, actionComplete);
+              break;
+            case 'detectFaces':
+              visualRecognition.detectFaces(params, actionComplete);
+              break;
+            case 'recognizeText':
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 6.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2713769514/issues/source?bid=3225115&fileBranchId=3344975#l236)